use_case_examples/hybrid_model/README.md

```diff
@@ -12,3 +12,13 @@ In this case we apply a fully connected layer of a GPT-2 model in FHE.
 1. Run FHE client using `python infer_hybrid_llm_generate.py`
    - You will first be asked about the number of tokens that you want to generate
    - Then you will be able to enter your prompt
+
+## Adapt to any LLM
+
+This use case can easily be used with a different model than GPT2 with any other model on Hugging Face.
+
+For example, you can use Phi 1.5 from microsoft and protect the weights that are used to compute Query, Keys, Values as follows:
```

Comment:
Maybe not mention Microsoft but just give a link to Phi 1.5 on HF or whatever side hosts it ?

Reply:
I gave a link to microsoft HF phi-1.5 model.